components/modal/index.en-US.md

```diff
@@ -62,6 +62,7 @@ The properties of the object are follows:
 
 | Property | Description | Type | Default |
 | -------- | ----------- | ---- | ------- |
+| autoFocusButton | Specify which button to autofocus | null\|string: `ok` `cancel` | `ok` |
```

Comment:
can you add something like this to the corresponding place in components/modal/index.zh-CN.md

```
| autoFocusButton | 指定自动获得焦点的按钮 | null\|string: `ok` `cancel` | `ok` |
```